core-bundle/tests/Command/MigrateCommandTest.php

```diff
@@ -439,7 +439,7 @@ public function testAbortsOnWrongServerVersion(string $format): void
 
         $command = $this->getCommand([], [], null, null, $connection);
         $tester = new CommandTester($command);
-        $errorMessage = 'Wrong database version configured, please set it to "8.0.29", currently set to "5.7.39"';
+        $errorMessage = "Wrong database version configured!\nYou have version 8.0.29 but the database connection is configured to 5.7.39.";
```

Comment:
```suggestion
        $errorMessage = "Wrong database version configured! You have version 8.0.29 but the database connection is configured to 5.7.39.";
```
Line breaks need to be normalized in the test.